src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -110,25 +110,27 @@ def download(
 
 
 def _get_files(
-    file_name: Union[str, pathlib.PurePath, list[Union[str, pathlib.PurePath]]]
+    file_name: Union[str, pathlib.PurePath, list[Union[str, pathlib.PurePath]]],
+    path: Optional[str],
 ):
-    path = "/home/runner/.local/share/ansys_fluent_core/examples"
-    if os.path.exists(path):
-        if isinstance(file_name, str):
-            files = [str(os.path.join(path, os.path.basename(file_name)))]
-        elif isinstance(file_name, pathlib.PurePath):
-            files = [str(os.path.join(path, file_name.name))]
-        elif isinstance(file_name, list):
-            files = [
-                str(os.path.join(path, os.path.basename(file))) for file in file_name
-            ]
-    else:
-        if isinstance(file_name, str):
-            files = [file_name]
-        elif isinstance(file_name, pathlib.PurePath):
-            files = [str(file_name)]
-        elif isinstance(file_name, list):
-            files = [str(file) for file in file_name]
+    if isinstance(file_name, str):
+        file_path_check = os.path.join(path, os.path.basename(file_name))
+        files = (
+            [str(file_path_check)] if os.path.isfile(file_path_check) else [file_name]
+        )
+    elif isinstance(file_name, pathlib.PurePath):
+        file_path_check = os.path.join(path, file_name.name)
+        files = [file_path_check] if os.path.isfile(file_path_check) else [file_name]
```

Comment:
As mentioned earlier, I think we can still put these two `if` clauses together (`str` and  `Path`), by for example preparing the path with `file_name = Path(file_name)`

Reply:
Done. Thank you.